vmdb/app/models/miq_request_workflow.rb

```diff
@@ -957,8 +957,7 @@ def get_ems_metadata_tree(src)
       @ems_xml_nodes = {}
```

Comment:
```rails_logger``` doesn't actually do anything, so I glossed over that, thanks. I'll just remove the line.